libnavigation-core/src/main/java/com/mapbox/navigation/core/SetRoutesInfo.kt

```diff
@@ -0,0 +1,79 @@
+package com.mapbox.navigation.core
+
+import com.mapbox.navigation.base.internal.CurrentIndicesSnapshot
+import com.mapbox.navigation.core.directions.session.RoutesExtra
+
+internal sealed class SetRoutesInfo(
+    @RoutesExtra.RoutesUpdateReason val reason: String,
+    val legIndex: Int,
+) {
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (javaClass != other?.javaClass) return false
+
+        other as SetRoutesInfo
+
+        if (reason != other.reason) return false
+        if (legIndex != other.legIndex) return false
+
+        return true
+    }
+
+    override fun hashCode(): Int {
+        var result = reason.hashCode()
+        result = 31 * result + legIndex
+        return result
+    }
+
+    override fun toString(): String {
+        return "SetRoutesInfo(reason='$reason', legIndex=$legIndex)"
```

Comment:
should be?
```suggestion
        return "SetRoutesInfo(reason=$reason, legIndex=$legIndex)"
```

Reply:
That's what AS generated. I removed it, but it will reappear on regeneration.